app/objects/c_goals.py

```diff
@@ -0,0 +1,24 @@
+from app.objects.secondclass.c_goal import Goal
+from app.utility.base_object import BaseObject
+
+
+class Goals(BaseObject):
+    def __init__(self, goal_list=None):
+        super().__init__()
+        self.to_fulfill = list()
+        if goal_list:
+            for goal in goal_list:
+                new_goal = Goal(target=goal.target, value=goal.value, count=goal.count, operator=goal.operator)
+                self.to_fulfill.append(new_goal)
+        else:
+            self.to_fulfill.append(Goal(target='exhaustion', value='complete'))
```

Comment:
Do we want exhaustion to be the default case?

Reply:
I'm going to argue that yes, we do. It's most similar to how caldera already works, so no change in baseline, and the alternative would be to find something specific to aim for in every operation, which I think would be a bit hard challenging.